tests/TESTS/LinkLoss/device/LinkLoss

```diff
@@ -0,0 +1 @@
+../../../../services/LinkLoss
```

Comment:
Adding symbolink link should be part of the bootstrap process.

Reply:
Done: 5d79fc36863aba377c739eaa4cf961b71c2bd937